integration_tests/test_multiarch_stage_packages.py

```diff
@@ -31,6 +31,6 @@ def test_bad_multiarch_stage_packages(self):
         self.assertThat(
             exception.output, Contains(
                 "Error downloading stage packages for part 'my-part': The "
-                "Ubuntu package 'hello:fake-arch' was not found.\n"
+                "package 'hello:fake-arch' was not found.\n"
```

Comment:
Is this an enforced convention? At least in DNF, pushing an arch works by using `pkg.arch` rather than `pkg:arch`.